app/src/main/java/org/oppia/app/utility/KeyBoardHelpher.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+
+class KeyBoardHelpher {
```

Comment:
typo in this name.

Also rename this to `KeyboardHelper`.

Also, add Kdocs for this class.

Reply:
Done